### ISSUE TITLE
[Usable – Make Code – About] There is no chevron or underline for multiple link controls under 'About' pop-up window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3623]
**User Experience:**
User will be misguided If any actionable control(link) doesn't have underline or chevron as they will skip the functionality of such link. Not only this, user will face disappointment if visually appearing link display as text as user won't be proceed further for relevant action.

**Note**: User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "About" option and select it.
7. Navigate through all the controls appearing under "About" dialog.
8. Verify visual 'Version: 3.0.17, 6.0.18, v2.2.0-rc6, Terms of Use & Privacy' link by using Developer tool(F12).

**Actual Result:**
There is no chevron or underline for 'Version: 3.0.17, 6.0.18, v2.2.0-rc6, Terms of Use & Privacy' link controls present on 'About' pop-up window and due to this, user is getting affected as he/she might consider that link as text and will skip the functionality of such link.

**Expected Result:**
There must be chevron or underline on visual 'Version: 3.0.17, 6.0.18, v2.2.0-rc6, Terms of Use & Privacy' link controls present on 'About' pop-up window, so that user could easily navigate that link to get more details.

**Note:**
This issue also exists with Firefox and Chrome Browser.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

![Usable_About_There is no underline](https://user-images.githubusercontent.com/72871089/98519951-74ce1100-2297-11eb-9c0d-fe8236c70e74.jpg)

**Comment by jwunderl** [comment]
Appears as link text with blue color / underline on hover / focus
